airmozilla/manage/templates/manage/picturegallery.html

```diff
@@ -16,56 +16,81 @@
 
 {% block site_css %}
   {{ super() }}
-  <style>
-  #total-pictures-stats {
-      float: right;
-      font-size: 80%;
-  }
-  div.filter-bar {
-      margin: 20px;
-  }
-  div.filter-bar input[type="text"] {
-      width: 400px;
-  }
-  .current-event-picture div {
-      background-color: lightblue;
-  }
-  div.thumbnail {
-      position: relative;
-  }
-  .picture-metadata {
-      position: absolute;
-      right: 5px;
-      top: 5px;
-      opacity: 0.9;
-  }
-  </style>
+  {% compress css %}
+  <link href="{{ static('manage/css/picture-gallery.css') }}" rel="stylesheet" type="text/css">
+  {% endcompress %}
+
 {% endblock %}
 
 {% block manage_content %}
 
-{% if event %}
-<h4>
-  You are here to associate a picture with
-  <a href="{{ url('manage:event_edit', event.id) }}">{{ event.title }}</a>
-
-  <script>
-  window.CURRENT_EVENT = {{ event.id }};
-  {% if event.picture %}
-  window.CURRENT_EVENT_PICTURE = {{ event.picture.id }};
-  {% endif %}
-  </script>
-
-</h4>
-
-{% endif %}
-
 <div ng-app="picturegalleryApp" ng-controller="PictureGalleryController">
   <p ng-if="loading" class="loading">
     <img src="{{ static('img/spinner.gif') }}">
     <span class="blinking">Loading pictures...</span>
   </p>
 
+  {% if event %}
+    <h4>
+      You are here to associate a picture with
+      <a href="{{ url('manage:event_edit', event.id) }}">{{ event.title }}</a>
+
+    <script>
+      window.CURRENT_EVENT = {{ event.id }};
+      {% if event.picture %}
+        window.CURRENT_EVENT_PICTURE = {{ event.picture.id }};
+      {% endif %}
+    </script>
+
+    </h4>
+
+    {% if event.placeholder_img %}
+
+      <p>
+        <a class="btn btn-danger"
+           ng-show="!deleteAllConfirmation"
+           ng-click="toggleDeleteAllConfirmation()">
+          <i class="glyphicon glyphicon-minus-sign"></i>
+          Remove all pictures for {{ event.title }}
```

Comment:
This could make the button really really long. 
![screenshot 2015-01-09 09 04 09](https://cloud.githubusercontent.com/assets/26739/5683655/81d013d4-97de-11e4-9a77-d02b47b4a61c.png)

Instead, let's change it to something like "Remove all pictures made for this event". What do you think?